linodecli/__init__.py

```diff
@@ -112,7 +109,7 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
         if not parsed.command:
             # print version info and exit - but only if no command was given
             print(f"linode-cli {VERSION}")
-            print(f"Built off spec version {cli.spec_version}")
+            print(f"Built from spec version {cli.spec_version}")
```

Comment:
This isn't really relevant to the rest of these changes, but the wording was bothering me a bit 🙂